install.sh

```diff
@@ -628,12 +628,21 @@ detect_init_system() {
 }
 
 choose_install_mode() {
+  echo -n 'Sandstorm makes it easy to run web apps on your own server. '
+
   if [ "yes" = "$USE_DEFAULTS" ] ; then
     CHOSEN_INSTALL_MODE="${CHOSEN_INSTALL_MODE:-2}"  # dev server mode by default
   fi
 
+  if [ "no" = "${PREFER_ROOT:-}" ] ; then
+    echo ""
+    echo "NOTE: Showing you all options, including development options, but omitting "
+    echo "      init script automation, because you chose to install without using root."
+    CHOSEN_INSTALL_MODE="${CHOSEN_INSTALL_MODE:-2}"  # dev server mode by default
```

Comment:
Please don't default to a dev server when not root!

It's totally legitimate for people to want to install a live server with sandcats and everything as non-root.